es6-arrow-functions/main.js

```diff
@@ -0,0 +1,45 @@
+const $jokeContainer = document.querySelector('#joke-container');
+const $flashImage = document.querySelector('#flash-image');
+const $jokeForm = document.querySelector('#joke-form');
+
+const jokester = {
+  tellJoke: function (setup, punchline) {
+    $jokeForm.classList.add('d-none');
+    const $introStatement = renderJokePhrase('Hey Flash...');
+    appendJokePhrase($introStatement);
+    setTimeout(function () {
+      const $jokeSetup = renderJokePhrase(setup);
+      appendJokePhrase($jokeSetup);
+      setTimeout(function () {
+        const $jokePunchline = renderJokePhrase(punchline);
+        appendJokePhrase($jokePunchline);
+        flash.laugh();
+      }, 2000);
+    }, 2000);
+  }
+
+};
+const renderJokePhrase = phrase => {
+  const $phrase = document.createElement('h4');
+  $phrase.textContent = phrase;
+  $phrase.className = 'fade-in text-center';
+  return $phrase;
+};
+const appendJokePhrase = $phrase => $jokeContainer.append($phrase);
```

Comment:
Instead of extracting the methods to free functions, try refactoring the code such that they remain methods, but the appropriate functions have been converted to arrow functions.